Few-iOS/TableView.swift

```diff
@@ -216,16 +275,46 @@ public class TableView: Element {
 					tableView.deselectRowAtIndexPath(tableSelected, animated: false)
 				}
 			}
+			if let header = header {
+				handler.headerView.updateWithElement(header)
+				if tableView.tableHeaderView != handler.headerView {
+					tableView.tableHeaderView = handler.headerView
+				}
+			} else if tableView.tableHeaderView == handler.headerView {
+				tableView.tableHeaderView = nil
+			}
+			if let footer = footer {
+				handler.footerView.updateWithElement(footer)
+				if tableView.tableFooterView != handler.footerView {
+					tableView.tableFooterView = handler.footerView
+				}
+			} else if tableView.tableFooterView == handler.footerView {
+				tableView.tableFooterView = nil
+			}
+
 		}
+
 	}
 	
 	public override func createView() -> ViewType {
-		let tableView = FewTableView(frame: CGRectZero)
-		tableView.handler = TableViewHandler(tableView: tableView, elements: elements, headers: headers, footers: footers)
+		let tableView = FewTableView(frame: UIScreen.mainScreen().bounds)
```

Comment:
I hope Few's layout system is more reliable than UIKit's, because doing that with table views always gives me jank